students/allen_maxwell/Lesson_03/slicing_lab.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+# Allen Maxwell
+# Python 210
+# 11/9/2019
+# slicing_lab.py
+
+# returns the string with the first and last items exchanged.
+def exchange_first_last(seq):
+    return seq[-1 : ] + seq[1 : -1] + seq[ : 1]
```

Comment:
Don't want whitespace in these instances, no spacing around colon here as well as the rest of the code.